figma-shop/src/app/pages/product-details/details.resolver.ts

```diff
@@ -14,8 +14,11 @@ export class ProdDetailsResolver implements Resolve<ProductInterface>{
   public resolve(route: ActivatedRouteSnapshot): Observable<any> | Promise<any> | any {​​
     const id = route.params.id;
     return new Promise((resolve, reject) => {
-      return this.prodService.getOneProduct(id)
-        .then((data: ProductInterface) => resolve(data))
+      Promise.all([
```

Comment:
Why do you need this to `Promise.all` ?
As I understand it right, 2 calls are not related to each other.

Reply:
Fix it